components/sign/ui-sign-input/SignPasswordConfirmInput.tsx

```diff
@@ -0,0 +1,84 @@
+import styles from "./SignInput.module.scss";
+import classNames from "classnames/bind";
+import { SignPasswordConfirmInputProps } from "./constant";
+import Image from "next/image";
+import { useState } from "react";
+
+const cx = classNames.bind(styles);
+
+export const SignPasswordConfirmInput = ({
+  values,
+  setValues,
+  passwordConfirmErrorMessageRef,
+  passwordConfirmErrorInputRef,
+  showPasswordConfirmError,
+  setShowPasswordConfirmError,
+}: SignPasswordConfirmInputProps) => {
+  const [showPassword, setShowPassword] = useState(false);
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setValues({ ...values, passwordConfirm: e.target.value });
+  };
+
+  const handleBlur = () => {
+    setShowPasswordConfirmError(true);
+
+    if (!values.passwordConfirm) {
+      passwordConfirmErrorMessageRef.current.textContent =
+        "비밀번호를 입력해주세요.";
+      return;
+    }
+
+    if (values.passwordConfirm !== values.password) {
+      passwordConfirmErrorMessageRef.current.textContent =
+        "비밀번호가 다릅니다.";
+      return;
+    }
+
+    if (values.passwordConfirm && values.passwordConfirm === values.password) {
+      return setShowPasswordConfirmError(false);
+    }
+  };
```

Comment:
```suggestion
const handleBlur = () => {
    // 에러 메시지 초기화
    let errorMessage = "";

    // 비밀번호 확인 입력값 검증
    if (!values.passwordConfirm) {
        errorMessage = "비밀번호를 입력해주세요.";
    } else if (values.passwordConfirm !== values.password) {
        errorMessage = "비밀번호가 다릅니다.";
    }

    // 에러 메시지 업데이트 및 에러 상태 관리
    if (errorMessage) {
        passwordConfirmErrorMessageRef.current.textContent = errorMessage;
        setShowPasswordConfirmError(true);
    } else {
        setShowPasswordConfirmError(false);
    }
};

```

이부분도 이렇게 수정할 수 있겠네여

마지막 조건은 비밀번호 컨펌 입력값 이랑, 비밀번호 입력값이랑 같은 상황이라 이미 앞에 조건식에서 판별했어서 마지막 if문은 없어도 되겟습니다.

그래서 에러메시지가 있는 경우. 는 비밀번호가 입력안됐거나, 비밀번호가 다른 경우이고, 에러메시지가 없으면 이상 없다는 의미 여서
setShowPasswordConfirmError를 false로 안보이게 합니다.